phpunit.xml.dist

```diff
@@ -1,53 +1,38 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <phpunit
-	colors="true"
-	processIsolation="false"
-	stopOnFailure="false"
-	bootstrap="./tests/bootstrap.php"
-	>
-	<php>
-		<ini name="memory_limit" value="-1"/>
-		<ini name="apc.enable_cli" value="1"/>
-	</php>
+    colors="true"
+    processIsolation="false"
+    stopOnFailure="false"
+    bootstrap="./tests/bootstrap.php"
+    >
+    <php>
+        <ini name="memory_limit" value="-1"/>
+    </php>
 
-	<!-- Add any additional test suites you want to run here -->
-	<testsuites>
-		<testsuite name="authentication">
-			<directory>tests/TestCase/</directory>
-		</testsuite>
-		<!-- Add plugin test suites here. -->
-	</testsuites>
+    <!-- Add any additional test suites you want to run here -->
+    <testsuites>
+        <testsuite name="Authentication">
```

Comment:
I recommend always using lowercase, dashed, as we do for all repos and is easier to type in cli.